libraries/shared/storage/backfill/storage_value_loader_test.go

```diff
@@ -151,7 +150,17 @@ var _ = Describe("StorageValueLoader", func() {
 		Expect(keysLookupOne.GetKeysCalled).To(BeTrue())
 		Expect(keysLookupTwo.GetKeysCalled).To(BeTrue())
 
-		Expect(bc.BatchGetStorageAtCalls).To(ContainElement(fakes.BatchGetStorageAtCall{BlockNumber: bigIntBlockOne, Account: addressTwo, Keys: manyKeys[0:backfill.MaxRequestSize]}))
+		found := false
+		for _, call := range bc.BatchGetStorageAtCalls {
```

Comment:
Is it possible to get rid of the loop here?

Reply:
I couldn't think of a simple way to do it but I may be lacking in imagination 😅